tests/experiments/experiment_unittests.py

```diff
@@ -340,17 +374,20 @@ def run_tests(raw_args=None, buffer=True, print_results=True):
         "--no-tests",
         required=False,
         action="store_true",
-        help="Only test the main experiments, not those in testing_archive/",
+        help="Only test the main experiments, not those in tests/",
+    )
+    parser.add_argument(
+        "--verbosity", required=False, default=1, help="Verbosity level, larger number increases verbosity."
     )
```

Comment:
```suggestion
        "--verbose", required=False, help="Print more information on tests run"
        "--quiet", required=False, help="Print less information on tests run"
    )
```
There's only three levels of verbosity (0, 1, 2) - having two extra arguments to adjust from the default level is nicer to use

Reply:
I've made this change, but added them as a mutually exclusive argument group so you can only specify one or the other, not both.

Reply:
Woops that's in a work-in-progress, have yet to push that commit